home/home3.1/inverse_calculator/src/test.rs

```diff
@@ -0,0 +1,33 @@
+use super::*;
+
+#[test]
+fn test_product_inverse() {
+    let x = 2.0;
+    let y = 3.0;
+    let z = 4.0;
+    assert_eq!(calculate_inverse(x, y, z), 1.0 / (x * y * z));
```

Comment:
minor: in general it is better to calculate exact expected value not formula. cause formula can be changed even with the same result of function on that input